components/Lsp2Coder/Lsp2Coder.tsx

```diff
@@ -146,13 +148,15 @@ const Lsp2Coder: React.FC = () => {
             url: decoded[0].value.url,
             hash: decoded[0].value.hash,
             hashFunction: 'keccak256(utf8)',
+            json: {},
           });
     } catch (error) {
       setDecodedValue('');
       setJsonUrlDecodedValue({
         url: '',
         hash: '',
```

Comment:
These fields don't exist anymore. They are now `verification?: { method: string, data: string, source?: string }`. Not sure if this is required in these tests, but during decoding you need to read the old and new values inside of LSP3 and LSP4 JSON, but the JSONURL parser will return the new structure always.

Reply:
Indeed! Let's update that in another PR.

Reply:
Oh i see you put a change request, then, let's do it in that pr haha:)

Reply:
I've put the record as <string, string> then, if data and source are always plain text.

What else to change? I was told that the methods/properties stay the exact same, just the types have to be updated.

Reply:
Please check:

- https://github.com/lukso-network/LIPs/commit/3872ecb96b6b7c247a445ebe008b67e299b4864a
- https://github.com/lukso-network/LIPs/commit/5ab051d3a40bff304586df62869378e97514e1eb

Before:

```js
{
    "hashFunction": 'keccak256(bytes)',
    "hash": 'string',
    "url": 'string',
    "fileType": 'string'
},
```


After:

```js
{ // example of a signature based avatar verification
    "verification": {
        "method": 'ecdsa',
        "data": 'string', // signer that signed the bytes of the file
        "source": 'string' // e.g url returning the signature of the signed file
    },
    "url": 'string',
    "fileType": 'string'
}
```